src/app/containers/ChartbeatAnalytics/index.jsx

```diff
@@ -41,6 +41,7 @@ const ChartbeatAnalytics = ({ data }) => {
     service,
     sendCanonicalChartbeatBeacon,
     isCanonicalAndEnabled,
+    chartbeatConfig,
```

Comment:
I think we could get rid of the hook dependencies that aren't been used here

Reply:
Any idea why they were there in the first place @rhenshaw56 , code been deleted but this not updated?

Reply:
@andrewscfc I think they were being used [when `getConfig` was called inside the hook](https://github.com/bbc/simorgh/commit/dabc85450d94e01d3ed849d6c7a414213caf6e0c#diff-f85d75438a01e2cb63e2f407f289dfc6) but that's been moved out to line 19

Reply:
@rhenshaw56 just found this change causes an infinite render loop!
<img width="673" alt="image" src="https://user-images.githubusercontent.com/9645462/69956509-18e00180-14f8-11ea-94e7-3e462835614b.png">

I'm not quite sure why yet, just looking into it. I may have to back out https://github.com/bbc/simorgh/pull/4861/commits/bd1eb2e871be6a813fc822a6c09790eeb8089610 and put in an eslint exception..

Reply:
Should be resolved by: 8a47c6aebbc6ec275c042fc0570431aa3b4c6bef

Reply:
As below, took resolved by e91c6d37ee509da4bc8b039dd5143aea2bb0c3b3